app/src/main/java/org/kiwix/kiwixmobile/downloader/DownloadService.java

```diff
@@ -218,6 +223,7 @@ public void toggleDownload (int notificationID) {
   }
 
   public void pauseDownload(int notificationID) {
+    Log.i("kiwixdownloadservice", "Pausing ZIM Download");
```

Comment:
As per my comment on line 183 - could be useful to know more about the download that's being paused and the state of that download.